### ISSUE TITLE
Choice of action on startup should be in Settings menu

### ISSUE THREAD

**Issue by ilyats** [koreader/koreader#6504]
**Does your feature request involve difficulty completing a task? Please describe.**
Now the action on startup (entering file browser, opening last document etc.) is chosen with "Start with:" submenu in the leftmost menu in file browser mode. I was unable to find it there without googling for solution, even though I have been using Koreader for years. 

**Describe the solution you'd like**
It should be in the Settings menu, possibly with submenu name like "Action at startup".

**Describe alternatives you've considered**
Another possible place is the leftmost menu (where "Exit" item is located).
**Additional context**
Add any additional context or screenshots about the feature request here.

**Comment by farvardin** [comment]
I agree, i was browsing the issue tracker because i was looking for this option and wasn't sure if it was available. I didn't even know this menu was available. There shouldn't be a contextual menu for this purpose, all options should always be available. The library menu doesn't seem to bring many more entries.
